spec/test.txt.orig

```diff
@@ -0,0 +1,5 @@
+<<<<<<< HEAD
+test
+=======
+test3
+>>>>>>> 693aeb97b8116b6ce2e176b3448589bd1dda16e9
```

Comment:
@glyphrider This file should not be added to github.  *.orig files should be added to the .gitignore file.